layouts/community/ergodox/bocaj/keymap.c

```diff
@@ -191,9 +155,35 @@ bool process_record_keymap(uint16_t keycode, keyrecord_t *record) {
       }
       return false;
       break;
+    case KC_MAKE:  // Compiles the firmware, and adds the flash command based on keyboard bootloader
+      if (!record->event.pressed) {
+        uint8_t temp_mod = get_mods();
+        clear_mods();
+        send_string_with_delay_P(PSTR("make " QMK_KEYBOARD ":" QMK_KEYMAP), 10);
+        if (temp_mod & MODS_SHIFT_MASK) {
+          #if defined(__ARM__)
+            send_string_with_delay_P(PSTR(":dfu-util"), 10);
+          #elif defined(BOOTLOADER_DFU)
+            send_string_with_delay_P(PSTR(":dfu"), 10);
+          #elif defined(BOOTLOADER_HALFKAY)
+            send_string_with_delay_P(PSTR(":teensy"), 10);
+          #elif defined(BOOTLOADER_CATERINA)
+            send_string_with_delay_P(PSTR(":avrdude"), 10);
+          #endif // bootloader options
+        }
+        #if defined(KEYBOARD_viterbi)
+          send_string_with_delay_P(PSTR(":dfu"), 10);
+        #endif
+        if (temp_mod & MODS_CTRL_MASK) { send_string_with_delay_P(PSTR(" -j8 --output-sync"), 10); }
+        send_string_with_delay_P(PSTR(SS_TAP(X_ENTER)), 10);
+        set_mods(temp_mod);
+        layer_move(default_layer);
```

Comment:
Actually, you should be able to move this to layer 0, regardless of the default layer.  This is because the default layer is on layer 0, always.